tests/integration/components/ice-tooltip-icon-test.js

```diff
@@ -20,7 +18,7 @@ test('target icon renders', async function(assert) {
     {{/ice-tooltip-icon}}
   `);
 
-  const icon = IconHelper.create();
+  let icon = IconHelper.create();
```

Comment:
In the future when working on a PR, if unrelated changes like this are necessary we should pull them out into a separate PR so there's less noise to review. The workflow would be to checkout from master, convert all `const` -> `let`, make and merge that PR, then rebase this PR on that one. Don't worry about it for this one though, better to get this in and do it that way from now on.